queries.sql

```diff
@@ -45,3 +45,81 @@ WHERE name <> 'Gabumon';
 SELECT *
 FROM animals
 WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+
+-- ---day 2 -------
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts, species)
+VALUES
+    ('Charmander', '2020-02-08', -11, false, 0, 'Fire'),
+    ('Plantmon', '2021-11-15', -5.7, true, 2, 'Grass'),
+    ('Squirtle', '1993-04-02', -12.13, false, 3, 'Water'),
+    ('Angemon', '2005-06-12', -45, true, 1, 'Angel'),
+    ('Boarmon', '2005-06-07', 20.4, true, 7, 'Mammal'),
+    ('Blossom', '1998-10-13', 17, true, 3, 'Plant'),
+    ('Ditto', '2022-05-14', 22, true, 4, 'Unknown');
+
+
+
```

Comment:
- [ ] I'd like to bring up an important point regarding our project requirements. It appears that the following tasks, which were initially outlined in the project requirements, have not been completed:

<img width="1006" alt="Screen Shot 2023-10-13 at 13 25 45" src="https://github.com/CobbyElsonfx/vet_clinic/assets/102757126/2bda6ac9-ba59-4981-a892-a2a471be2821">

Reply:
Thanks for the correction, Will update the table as soon as possible